data/src/main/java/com/example/data/database/entities/MainDB.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.data.database.entities
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.example.data.Utils.ConstantUtils
+
+@Entity
+data class MainDB(
+    @PrimaryKey(autoGenerate = true)
+    @ColumnInfo(name = "temperature") val temperature: Double = ConstantUtils.DEFAULT_DOUBLE,
+    @ColumnInfo(name = "feels_Like") val feelsLike: Double = ConstantUtils.DEFAULT_DOUBLE,
+    @ColumnInfo(name = "humidity") val humidity: Int = ConstantUtils.DEFAULT_INT,
+    @ColumnInfo(name = "min") val minimum: Double = ConstantUtils.DEFAULT_DOUBLE,
+    @ColumnInfo(name = "max") val maximum: Double = ConstantUtils.DEFAULT_DOUBLE,
+    @ColumnInfo(name = "pressure") val pressure: Int = ConstantUtils.DEFAULT_INT,
+    @ColumnInfo(name = "sea_level") val seaLevel: Int = ConstantUtils.DEFAULT_INT,
+    @ColumnInfo(name = "ground_level") val groundLevel: Int = ConstantUtils.DEFAULT_INT,
+    @ColumnInfo(name = "tempKf") val tempKf: Double = ConstantUtils.DEFAULT_DOUBLE
```

Comment:
what is kf?

Reply:
I do not know it well, it is an internal parameter according to the description, dont investigate more